modules/admin/proc-install-operator.adoc

```diff
@@ -19,23 +23,32 @@ Containers are available for the following CPU architectures:
 * You are logged in as an administrator on the {ocp-short} web console.
 * You have configured the appropriate roles and permissions within your project to create an application. For more information, see the link:https://docs.openshift.com/container-platform/{ocp-version}/applications/index.html[Red Hat OpenShift documentation on Building applications].
 
-.Procedure
+[NOTE]
+====
+For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the operator resources through role bindings or cluster role bindings. You can choose to deploy the operator in the `openshift-operators` namespace instead, however, the {product} operator shares the namespace with other operators, and therefore any users who can create workloads in that namespace can get their privileges escalated from all Operators' service accounts.
```

Comment:
```suggestion
For enhanced security, deploy the {product} Operator in a dedicated default namespace such as `rhdh-operator`. The cluster administrator can restrict other users' access to the operator resources through role bindings or cluster role bindings. Should you choose to deploy the operator in the default `openshift-operators` namespace, be advised that doing so could allow any users who can create workloads in that namespace can get their privileges escalated from all Operators' service accounts. This is generally not recommended due to the security risk. 
```

Reply:
Similarly to a [previous PR we had](https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/253/files), if something isn't recommended, especially for security reasons, we should either say "do not do X" or just remove mentioning it in the docs. I am removing this line.